src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTab, activeTabId, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={cn({
+              'is-active': activeTabId === tab.id,
+            })}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (activeTabId !== tab.id) {
+                  onTabSelected(tab.id);
+                }
+              }}
+            >
+              {tab.title}
+            </a>
+          </li>
```

Comment:
When rendering lists in React, each element should have a unique `key` prop. This helps React identify which items have changed, been added, or removed. Please add a `key` prop to each `<li>` element, using a unique identifier like `tab.id`.